internal/util/util.go

```diff
@@ -224,6 +224,12 @@ func IsWordChar(r rune) bool {
 	return unicode.IsLetter(r) || unicode.IsNumber(r) || r == '_'
 }
 
+// IsNonWordChar returns whether or not the string is not a 'word character'
+// Non word characters are defined as all characters not being alphanumerics nor delimiters
```

Comment:
nit: It would be good to keep this sentence in line with the corresponding sentence in `IsWordChar()` description.
Maybe just replace this sentence with `See IsWordChar`?

Reply:
Looks good to me, except that I still don't think that changing `'_'` to `delimiters` is a good idea. It makes it unclear and ambiguous. What delimiters?

So, to make it clear, we should at least say "subword delimiters" instead of just "delimiters"?

(Frankly, I would prefer to keep it simple and just leave `' '`. But since you introduce this "more general" notion of subword delimiter (BTW I'm still not sure for what exactly), well, let's at least do that consistently...)

Hmm, now I see you are using just "delimiters" instead of "subword delimiters" in multiple other function descriptions as well...

Reply:
> So, to make it clear, we should at least say "subword delimiters" instead of just "delimiters"?

I don't mind changing the wording "delimiters" to "subword delimiters".

> (Frankly, I would prefer to keep it simple and just leave '_'. But since you introduce this "more general" notion of subword delimiter (BTW I'm still not sure for what exactly), well, let's at least do that consistently...)

Try word and subword jumps on some _lisp_ code. When using `_` as a subword delimiter there is almost no difference between subword jumps and word jumps. When using `-` as subword delimiter in _lisp_ it works as it would for _go_ using `_` as a subword delimiter. Or compare the double click behavior when clicking e.g. `remove-if-not` ...

```lisp
 (defun csg-intersection-intersect-all (obj-a obj-b)
   (lambda (ray)
     (flet ((inside-p (obj) (lambda (d) (inside-p obj (ray-point ray d)))))
       (merge 'fvector
              (remove-if-not (inside-p obj-b) (intersect-all obj-a ray))
              (remove-if-not (inside-p obj-a) (intersect-all obj-b ray))
              #'<))))
```
So, subword delimiters are language dependent. We are just a little bit spoiled by the majority of the languages using just '_'.

There was this plan to make this a language dependent option. A little bit like:

```go
// Warning, super crappy code just for illustration purposes ...
delimiters := []rune(b.Settings["subword-delimiter"].(string))

func IsSubwordDelimiter(r rune, delimiters string) bool {
	return strings.ContainsRune(delimiters, r)
}

```

> Hmm, now I see you are using just "delimiters" instead of "subword delimiters" in multiple other function descriptions as well...

There are 4 references of "delimiters":
- `IsWordChar()`
- `IsNonWordChar()`
- `IsUpperWordChar()`
- `IsLowerWordChar()`

And they all deal with the question, if a rune belongs to a word. To answer that question, you need to take into account what subword delimiter you are using... For _go_ and _python_ this would be '-' and for _lisp_ '_'.
What is the issue with that?

Reply:
Nothing's wrong with language-dependent wordchars, it's just not clear why need to generalize the code now, before we even decided to implement this feature and decided how exactly it would look like. But as I said, I'm fine with generalizing it now, my minor remark was only that "delimiters" (not "subword delimiters") sounds unclear and ambiguous, nothing more.

Reply:
Okay, sounds good to me. "subword delimiters" it is.